src/Microsoft.Health.Dicom.Core/Modules/ServiceModule.cs

```diff
@@ -110,7 +111,7 @@ public void Load(IServiceCollection services)
                 .AsImplementedInterfaces();
 
             services.Add<DicomTagParser>()
-                         .Scoped()
```

Comment:
nit: can you reduce the indent here to only 1 level?

```csharp
services.Add<DicomTagParser>()
    .Singleton()
    .AsSelf()
    .AsImplementedInterfaces();
```

Reply:
If we use the same pattern (of `AsSelf()` and `AsImplementedInterfaces()`) everywhere, then it's probably worth a conversation with the team about how we declare these services. So I would leave this with your previous usage of:

```csharp
services.Add<DicomTagParser>()
    .Singleton()
    .AsSelf()
    .AsImplementedInterfaces();
```

If we want to change it, then we can change all of them